server.go

```diff
@@ -177,22 +177,22 @@ func WithTemplateFS(fs fs.FS) func(*Server) {
 	return func(c *Server) { c.fs = fs }
 }
 
-// WithCorsMiddleware registers a middleware to handle CORS.
+// WithGlobalMiddleware registers a middleware to handle CORS.
 // It is not handled like other middlewares with [Use] because it applies routes that are not registered.
 // For example:
 //
 //	import "github.com/rs/cors"
 //
 //	s := fuego.NewServer(
-//		WithCorsMiddleware(cors.New(cors.Options{
+//		WithGlobalMiddleware(cors.New(cors.Options{
 //			AllowedOrigins:   []string{"*"},
 //			AllowedMethods:   []string{"GET", "POST", "PUT", "DELETE", "OPTIONS"},
 //			AllowedHeaders:   []string{"*"},
 //			AllowCredentials: true,
 //		}).Handler)
 //	)
-func WithCorsMiddleware(corsMiddleware func(http.Handler) http.Handler) func(*Server) {
```

Comment:
Mind changing your commit to `BREAKING: ...`. So the changelog is more explicit on the change?